### PR TITLE
004-temporal revisions

### PR DESCRIPTION
**Pull request by amilan17**

### CODE REVIEW

**Comment by amilan17** [comment]
For review: 

*    [ ]  removal of XML examples that are duplicative of existing guidance. 

For discussion: 

*    [ ]  Why is there a distinct test for presence of temporal elements? Is it to give a score just for the existence of anything time related?

**Comment by amilan17** [comment]
keep test for temporal extent (no s)

File: kpi/004-temporal-information.adoc [hunk 1]
```diff
@@ -19,94 +17,46 @@ covered by products and how often new products are received.
 |===
 |Rule |Score
 
-|The Temporal Extent is present
+|The temporal `extent` section is present.
```
Comment (tomkralidis) [inline kpi/004-temporal-information.adoc:20]: The `gmd:temporalExtent` element is present

File: kpi/004-temporal-information.adoc [hunk 2]
```diff
@@ -19,94 +17,46 @@ covered by products and how often new products are received.
...
 |1
 
-|The begin and end components are present in the temporal extent
+|The `beginPosition` and `endPosition` elements are present in the temporal extent.
```
Comment (tomkralidis) [inline kpi/004-temporal-information.adoc:23]: `gml:beginPosition` && `gml:endPosition`

File: kpi/004-temporal-information.adoc [hunk 3]
```diff
@@ -19,94 +17,46 @@ covered by products and how often new products are received.
...
 |1
 
-|The begin date time is less than or equal to the end date time
+|The begin date time is less than or equal to the end date time.
 |1
 
-|The Data Update Frequency is present
+|The `maintenanceAndUpdateFrequency` elements are present.
```
Comment (tomkralidis) [inline kpi/004-temporal-information.adoc:29]: `gmd:maintenanceAndUpdateFrequency`

File: kpi/004-temporal-information.adoc [hunk 4]
```diff
@@ -19,94 +17,46 @@ covered by products and how often new products are received.
...
 |1
 
-|The Data Status is present
+|The `status` element is present.
```
Comment (tomkralidis) [inline kpi/004-temporal-information.adoc:32]: `gmd:status`

**Review by tomkralidis — CHANGES_REQUESTED** [review]

**Pull request merged**